src/index.html

```diff
@@ -5,9 +5,43 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="container">
+      <img
+        src="images/imac.jpeg"
+        alt="monitor"
+        class="monitor"
+      >
+
+      <div class="description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+
+      <div class="code">Product code: 195434</div>
+
+      <div class="rating">
+        <div class="stars">
```

Comment:
Add the modifier from the previous task
```suggestion
        <div class="stars stars--4">
```